services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -261,6 +262,17 @@ public MongoDbBasedRegistrationConfigProperties serviceProperties() {
         return new MongoDbBasedRegistrationConfigProperties();
     }
 
+    /**
+     * Gets properties for the hono tenant mongodb service.
+     *
+     * @return The properties.
+     */
+    @Bean
+    @ConfigurationProperties(prefix = "hono.tenant.mongodb")
```

Comment:
This should be `@ConfigurationProperties(prefix = "hono.tenant.svc")`

Reply:
resolved